### DISCUSSION TITLE
Player take kit message

### DISCUSSION THREAD

**Discussion by MaxWainer** [Zrips/CMI#3679] [Feature Requests]
**Description of feature/improvement request:**
When player take kit, send message about it. 
Like: 
```
playerTakeKit: '&eYou took the kit (&6[kitName]&e)!'
playerTakeKitMoney: '&eYou took the kit (&6[kitName]&e) for &6[kitPrice]!'
```
---
### Detailed explanation of what it should do, when and in what situation: ###
Can help to send cool messages with third party GUIs.
### Basic explanation as to why this feature is needed and/or useful to have: ###
Little function which can help to give kit from third party GUIs and with cool messages.

**Comment by MaxWainer** [comment]
Also you can add feature with providing displayname to kits

**Comment by CloudeLecaw** [comment]
You know you can create this allready with a simple customalias? created with CMI

to prevent using PlayerDisplayName to kits use playerName
